### PR TITLE
Only show signatures from account

### PR DESCRIPTION
**Pull request by danielfriis**

### CODE REVIEW

File: app/controllers/signatures_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class SignaturesController < ApplicationController
   before_action :authorize_template, only: [:create, :update, :destroy]
 
   def index
-    @ransack = @signatures.ransack(params[:q])
+    @ransack = @account.signatures.ransack(params[:q])
```
Comment (kaspernj) [inline app/controllers/signatures_controller.rb]: I believe the correct way to do this, would be to do this in the top of the controller instead:

```ruby
load_resource :account
load_and_authorize_resource :signature, through: :account
```

And then let it be:
```ruby
@ransack = @signatures.ransack(params[:q])
```

What do you think?

**Comment by kaspernj** [comment]
It seems like you have added a file to the repository, which I assume you didn't mean to? Maybe add that directory to `.gitignore`?

File: app/controllers/signatures_controller.rb (hunk 1 above)
Comment (danielfriis) [inline app/controllers/signatures_controller.rb]: But doesn't that require `account_id` on `signatures`?

Comment (kaspernj) [inline app/controllers/signatures_controller.rb]: @danielfriis It requires the relationship to exist.

Comment (danielfriis) [inline app/controllers/signatures_controller.rb]: Which isn't the case in this instance. Only through `template`, but that's not in the URL. So, has I taken the right approach you think?

Comment (kaspernj) [inline app/controllers/signatures_controller.rb]: Yeah. Then it is the right approach :-)

Comment (danielfriis) [inline app/controllers/signatures_controller.rb]: Phew! :D

Comment (danielfriis) [inline app/controllers/signatures_controller.rb]: I always get nervous when you comment on my stuff :P

**Pull request merged**